agreements.md

```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
 If the offensive comment is seen as severe and above the grade of this agreement, it will be raised to the HR circle to decide on further steps.
 Fortunately, so far, there have been no such situations.
 
+## Constructive feedback
+
+We aim to maintain a positive work environment where feedback is given constructively to promote growth and improvement.
+To manage situations where feedback might come across as negative or aggressive, we will use the following procedure:
+
+- If someone feels that feedback is too negative or aggressive towards any staff member, they should say:
```

Comment:
```suggestion
To manage situations where feedback may seem nonconstructive, we will use the following procedure:

- If someone feels that feedback is overly negative or aggressive towards any staff member, they should say:
```

negative feedback can be just constructive, rephrased a little to make it clearer we are concerned about non-constructive criticism